io/v6/server.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+type PlayerStore interface {
+	GetPlayerScore(name string) int
+	RecordWin(name string)
+	GetLeague() League
+}
+
+/* type ReadSeeker interface {
+	Reader
+	Seeker
+} */
+type Player struct {
+	Name string
+	Wins int
+}
+
+type PlayerServer struct {
+	store PlayerStore
+	http.Handler
+}
+
+const jsonContentType = "application/json"
+
+func NewPlayerServer(store PlayerStore) *PlayerServer {
+	p := new(PlayerServer)
+
+	p.store = store
+
```

Comment:
Ideally should should write a piece of comment on what this block does. Do practice adding comments to code blocks.